### PR TITLE
Contact Us Page > Form and Get In Touch Header

### PR DESCRIPTION
**Pull request by hanlinc27**
### Description
Contact Us Page header and form rough layout. I elected to use styled components because I felt it was simpler to reference for a component-heavy page. Will definitely clean up the repetitive styling in a future PR 
[Figma Link
](https://www.figma.com/file/n0hwxK89ZLLlYH8gro0iub/Accelerator-Project?node-id=3176%3A152)
[Trello Link
](https://trello.com/c/CyLoXLUC)
### Demo

ezgif com-gif-maker
)

### Next Steps
* Fix up the styling for the components, most notably the Grid Container, Radio Buttons, and Preview on the Drop Down
* Add in state handling for the form components (just simple React hook) for form input values 
* Check for correct input (regex)
* Discussion on where we want the data to be stored --> Maybe just connect to an Airtable API?

### CODE REVIEW

**Comment by hanlinc27** [comment]
Oops the file diff is caused by running prettier

File: gatsby-config.js [hunk 1]
```diff
@@ -5,32 +5,33 @@
...
-            },
+  /* Your site config here */
+  plugins: [
+    {
+      resolve: `gatsby-plugin-google-fonts`,
+      options: {
+        fonts: [
+          `Josefin Sans\:300,400,500,600,700,800,900`,
+          `Poppins\:300,400,500,600,700,800,900`,
+          `Poppins\:300,400,500,600,700,800,900`,
```
Comment (BBoutet1) [inline gatsby-config.js:16]: Please can you remove the second "Poppins" line

**Comment by hanlinc27** [comment]
I'm going to try to undo the linting on all the files and push up new changes because the diff is way too messy right now. The relevant page should be `ContactUsForm.js` though if anyone would like to take a look.

File: gatsby-config.js (hunk 1 above)
Comment (BBoutet1) [inline gatsby-config.js]:
```
          `Poppins\:300,400,500,600,700,800,900`,
```
Please @hanlinc27  can you remove the duplicate line?

**Review by BBoutet1 — CHANGES_REQUESTED** [review]

**Review by BBoutet1 — DISMISSED** [review]

**Comment by BBoutet1** [comment]
@hanlinc27 you'll need to fix the conflicts as "Volunteer/index.js" has been modified in the previous PR.

**Comment by dghazi12** [comment]
Just a couple of minor things - First, the cheetoh positioning is off/missing a cheetoh as per Figma. There is also a typo in the "fill in the form below.... " sentence. The word missing is "touch" 

PR: 

Screen Shot 2020-12-06 at 8 48 20 PM

FIGMA : 

Screen Shot 2020-12-06 at 8 44 45 PM

Otherwise, looks pretty good!!

**Pull request merged**